src/main/java/com/sap/sgs/phosphor/fosstars/model/feature/oss/OssFeatures.java

```diff
@@ -199,4 +199,10 @@ private OssFeatures() {
    */
   public static final Feature<LgtmGrade> WORSE_LGTM_GRADE = new LgtmGradeFeature(
       "The worse LGTM grade of a project");
+
+  /**
+   * Shows if an open-source project uses verified signed commits.
```

Comment:
Could you please add a link to the GitHub docs?

```
See <a link="https://help.github.com/en/github/authenticating-to-github/managing-commit-signature-verification">Managing commit signature verification</a> on GitHub.
```

Reply:
Added the attribute `@see`